project_euler/problem_50/sol1.py

```diff
@@ -0,0 +1,102 @@
+"""
+Consecutive prime sum
+
+Problem 50
+
+The prime 41, can be written as the sum of six consecutive primes:
+
+41 = 2 + 3 + 5 + 7 + 11 + 13
+
+This is the longest sum of consecutive primes that adds to
+a prime below one-hundred.
+
+The longest sum of consecutive primes below one-thousand that adds to a prime,
+contains 21 terms, and is equal to 953.
+
+Which prime, below one-million, can be written as the sum of
+the most consecutive primes?
+
+Solution:
+
+First of all, we need to generate all prime numbers
+from 2 to the closest prime number with 1000000.
+Then, use sliding window to get the answer.
+"""
+
+from math import floor, sqrt
+
+
+def is_prime(number: int) -> bool:
+    """
+    function to check whether a number is a prime or not.
+    >>> is_prime(2)
+    True
+    >>> is_prime(6)
+    False
+    >>> is_prime(1)
+    False
+    >>> is_prime(-7000)
+    False
+    >>> is_prime(104729)
+    True
+    """
+
+    if number < 2:
+        return False
+
+    for n in range(2, floor(sqrt(number)) + 1):
+        if number % n == 0:
+            return False
+
+    return True
+
+
+def solution():
+    """
+    Return the problem solution.
+    >>> solution()
+    997651
```

Comment:
Please don't put this in your doctest as it is already being tested by this [script](https://github.com/TheAlgorithms/Python/blob/master/project_euler/validate_solutions.py).

If possible put the default argument value as `10 ** 6` and have `doctest` for the lower number input value to test the function.

Reply:
ok

Reply:
So I only need to remove that test?

Reply:
Include a positional argument for the function and set the default value as the question input which is probably `10 ** 6` and then tests your function with other input values and not with the default one.

We want to test the function and not the answer because the answer is tested by us.

Reply:
Alright. I'll do that soon